test/unit/master/test_cluster_runner_config.py

```diff
@@ -55,6 +55,15 @@ class TestClusterRunnerConfig(BaseUnitTestCase):
     PHPUnit:
     """
 
+    _VALID_CONFIG_WITH_EMPTY_COMMAND = """
+    PHPUnit:
+        commands:
+            - echo "go"
+            -
```

Comment:
You're absolutely right. I left it out earlier because I thought it's not necessary to (sort of) test PyYAML here. But you're totally right that we want to keep the behavior consistent even we choose not to use PyYAML in the future. Will add.